ts/src/program/namespace/account.ts

```diff
@@ -227,6 +230,62 @@ export class AccountClient<
     });
   }
 
+  /**
+   * Returns an `EventEmitter` emitting a "change" event whenever any program account
+   * of <T> changes.
+   */
+  subscribeAll(commitment?: Commitment): EventEmitter {
+    if (programAccountSubscription !== null) {
+      return programAccountSubscription.ee;
+    }
+
+    const ee = new EventEmitter();
+    const listener = this._provider.connection.onProgramAccountChange(
+      this._programId,
+      async (info: KeyedAccountInfo) => {
+        const account = this._coder.accounts.decode(
+          this._idlAccount.name,
+          info.accountInfo.data
+        );
+        ee.emit("change", {
+          publicKey: info.accountId,
+          account,
+        });
+      },
+      commitment,
+      [
+        {
+          memcmp: {
```

Comment:
This should be `this.coder.memcmp(this._idlAcocunt.name),`. The program client no longer assumes a borsh coder.